client/client.go

```diff
@@ -147,6 +147,20 @@ func (c *Client) Init(rootKeys []*data.PublicKey, threshold int) error {
 	return c.local.SetMeta("root.json", rootJSON)
 }
 
+// InitWithRoot initializes a local repository from root metadata.
```

Comment:
Very small nit: can we call out that this explicitly does not try to fetch or update from a remote?

Reply:
Added!